doc/release-notes.md

```diff
@@ -1,6 +1,23 @@
 # Release 0.14.13
```

Comment:
@andrewpmartinez Release 0.15.0?

Reply:
@andrewpmartinez i have a pr to this branch out there that updates the readme to 0.15.0 as edward points out. also shouldn't the databaser snapshot stuff get put into 0.15.0?